packages/edit-site/src/components/secondary-sidebar/inserter-sidebar.js

```diff
@@ -18,7 +18,8 @@ import { useEffect, useRef } from '@wordpress/element';
 import { store as editSiteStore } from '../../store';
 
 export default function InserterSidebar() {
-	const { setIsInserterOpened } = useDispatch( editSiteStore );
+	const { closeGeneralSidebar, setIsInserterOpened } =
+		useDispatch( editSiteStore );
```

Comment:
is it worth also grabbing `isGeneralSidebarOpen` from the below `useSelect` and add something like:

```js
function handleOnCategorySelection() {
    if ( isGeneralSidebarOpen ) {
        closeGeneralSidebar();
    }
}
```
to save firing closeGeneralSidebar with every category selection if we don't need to?